index.js

```diff
@@ -622,6 +638,48 @@ var QUERIES = {
       }, [])
     }
   },
+  baseline: {
+    matches: ['baseline'],
+    regexp: /^baseline\s+(\d+|high|low)$/i,
```

Comment:
`high` and `low` terms are official? They look a little cryptic for me.

Reply:
Ah, yes, `high` and `low` are more internal terms used by the community group.

Should be `widely available` and `newly available` to align with public docs on pages likes MDN: https://developer.mozilla.org/en-US/blog/baseline-evolution-on-mdn/#refined_definition

Reply:
> Should be widely available and newly available to align with public docs

Like it more